src/renderers/native/ReactNativeFiber.js

```diff
@@ -416,4 +417,11 @@ const ReactNative = {
 
 };
 
+if (typeof injectInternals === 'function') {
```

Comment:
Do you mean always exporting those functions and checking `typeof __REACT_DEVTOOLS_GLOBAL_HOOK__ !== 'undefined' &&
  __REACT_DEVTOOLS_GLOBAL_HOOK__.supportsFiber` inside of them? Or doing the check once but exporting a noop?